blueprints/gke/patterns/redis-cluster/main.tf

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2023 Google LLC
```

Comment:
2024

Reply:
because of this, couldn't find any other way gavinbunney/terraform-provider-kubectl#226